scripts/builds/react-router-dom.js

```diff
@@ -68,14 +62,8 @@ const webModules = [
         presets: ['@babel/preset-modules', '@babel/preset-react'],
         plugins: ['babel-plugin-dev-expression']
       }),
-      replace({ 'process.env.NODE_ENV': JSON.stringify('development') })
-      // TODO: Closure compiler complains about duplicate identifiers for stuff
-      // react-router-dom uses that it also re-exports from react-router
-      // compiler({
-      //   compilation_level: 'SIMPLE_OPTIMIZATIONS',
-      //   language_in: 'ECMASCRIPT_2018',
-      //   language_out: 'ECMASCRIPT_2017'
-      // })
+      replace({ 'process.env.NODE_ENV': JSON.stringify('development') }),
+      compiler()
```

Comment:
This is the development bundle, so we actually don't want this compressed. It should remain developer-readable.

Reply:
Looks like the next step below uses a `PRETTY` flag to apply `prettier` on the result.

Do you also want to avoid using the compiler here?

Reply:
Yeeeah, I'm really confused by this config. Why go through the rigamarole of prettying the code as you write it, transpiling, compressing, and re-prettying? That seems like a lot of work for practically unreadable code.

Reply:
I'm not really using closure compiler for minification, although it does that too. The long-term goal is to eventually run everything through closure compiler with advanced optimizations which will actually rewrite portions of the code. If people want the source they shouldn't be looking through node_modules anyway. That's what sourcemaps are for.